tla-bmcmt/src/main/scala/at/forsyte/apalache/tla/bmcmt/rules/FunCtorRule.scala

```diff
@@ -57,7 +57,7 @@ class FunCtorRule(rewriter: SymbStateRewriter) extends RewritingRule {
       .typed(TupT1(funT1.arg, funT1.res))
 
     val (afterMapState: SymbState, relationCells: Seq[ArenaCell]) =
-      mapCells(nextState, pairEx, varName, setEx, domainCells)
+      mapCells(nextState, pairEx, varName, domainCells)
```

Comment:
As a general comment, I'd prefer to see refactorings in an incremental, separate commit. Much easier to review.